### PR TITLE
Tutorial updates

### PR DESCRIPTION
**Pull request by Olajumoke01**
Updated version of the airbnb tutorial with missing code and commands. I think all the code in the tutorial should be working now. I added comments to the ssg manual edit file, to make it clearer which lines were added.

### CODE REVIEW

File: docs/source/tutorials/airbnb.rst [hunk 1]
```diff
@@ -337,7 +349,15 @@ The custom generator is the function ``airbnb_generators.user_age_provider``  (l
         sigma: float = query_results[0][1]
         return random.gauss(mu, sigma)
 
-Thus function returns a random age value based on the query results stored in the ``src-stats.yaml`` file. Using ``config.yaml`` over the AirBnb dataset, ``src-stats.yaml`` contains the following:
+After ``airbnb_generators.py`` has been edited, you need to generate an ``src-stats.yaml`` file by running:  ::
+
+    $ sqlsynthgen make-stats --config-file config.yaml --force
+
+This will generate a file with the results of your sql queries from config.yaml and these results will be stored in the variable ``SRC_STATS``.
```
Comment (cptanalatriste) [inline docs/source/tutorials/airbnb.rst:356]: I would reword this. Something like: "With this command, the results of the query defined in `config.yaml` will be stored in the `src-stats.yaml` file. If you recreate the generators, stored in `ssg.py`, you can verify that the query results are exposed via the `SRC_STATS` variable.

File: docs/source/tutorials/airbnb.rst [hunk 2]
```diff
@@ -337,7 +349,15 @@ The custom generator is the function ``airbnb_generators.user_age_provider``  (l
...
+From there they can be passed to your generators as arguments.
+
+In this example ``user_age_provider``, the ``query_results`` argument represents the results of the ``age_stats`` query and returns a random age value.
```
Comment (cptanalatriste) [inline docs/source/tutorials/airbnb.rst:359]: Just for clarity, I would reword it like: "Within the function `user_age_provider`, defined in `airbnb_generators.py`, the ..."

File: pyproject.toml [hunk 3]
```diff
@@ -25,6 +25,7 @@ sqlacodegen = "3.0.0rc1"
 asyncpg = "^0.27.0"
 greenlet = "^2.0.2"
 pymysql = "^1.1.0"
+pre-commit = {version = "^3.3.3", extras = ["dev"]}
```
Comment (cptanalatriste) [inline pyproject.toml:28]: Why was this dependency added?

**Comment by cptanalatriste** [comment]
@Olajumoke01 is looking good: I just made some minor suggestions. Also, if you still have them, it would be nice to add the complete `config.yaml` and `airbnb_generators.py` files to the tutorial.

File: pyproject.toml (hunk 3 above)
Comment (mhauru) [reply pyproject.toml:28]: I asked Olajumoke to add it because we use `pre-commit` checks and have a config file for it. This came up when I suggested that she run `pre-commit install`, but she needed to install pre-commit first. It's unrelated to any of the other changes in this PR, just came up at the same time.

**Review by cptanalatriste — CHANGES_REQUESTED** [review]
Some minor changes, that I flagged as comments.

File: docs/source/tutorials/airbnb.rst [hunk 4]
```diff
@@ -451,6 +473,12 @@ To use and get the most from story generators, we will need to make some changes
      - name: airbnb_generators.sessions_story
        num_stories_per_pass: 30  # see 3 below
 
+After editing the ``config.yaml`` file, you can run: ::
```
Comment (Iain-S) [inline docs/source/tutorials/airbnb.rst:476]: This all looks good to me. Out of curiosity, how (if at all) does this `::` method of code block differ from the kind we've been using? I.e.

```text
.. code-block::
```

Are they equivalent or is one better than the other?

Comment (mhauru) [reply docs/source/tutorials/airbnb.rst:476]: I _think_ they might be equivalent, `code-block` just gives you the opportunity to specify the language to use for syntax highlighting purposes. Not confident in this claim though.

File: pyproject.toml (hunk 3 above)
Comment (cptanalatriste) [reply pyproject.toml:28]: Makes sense :)

**Comment by Olajumoke01** [comment]
> @Olajumoke01 is looking good: I just made some minor suggestions. Also, if you still have them, it would be nice to add the complete `config.yaml` and `airbnb_generators.py` files to the tutorial.

Hey Carlos, Do you mean add them near to the end of the tutorial as an example or as separate files somewhere?

**Comment by Olajumoke01** [comment]
> @Olajumoke01 is looking good: I just made some minor suggestions. Also, if you still have them, it would be nice to add the complete `config.yaml` and `airbnb_generators.py` files to the tutorial.

Hey @cptanalatrist, Do you mean add them near to the end of the tutorial as an example or as separate files somewhere?

**Comment by cptanalatriste** [comment]
> > @Olajumoke01 is looking good: I just made some minor suggestions. Also, if you still have them, it would be nice to add the complete `config.yaml` and `airbnb_generators.py` files to the tutorial.
> 
> Hey @cptanalatrist, Do you mean add them near to the end of the tutorial as an example or as separate files somewhere?

Maybe you can add the files to the `airbnb` folder, and later add a paragraph at the end of the tutorial that goes: ``The complete files generated as part of this tutorial are available at [this location](htttp://your.link).

**Review by cptanalatriste — APPROVED** [review]
Looks good to me! Only a minor renaming request.

**Comment by Iain-S** [comment]
only one thing:

- [ ] move pre-commit dep from an extra to the dev group
